saagieapi/jobs/jobs.py

```diff
@@ -892,3 +892,83 @@ def export(
             logging.warning("❌ Job [%s] has not been successfully exported", job_id)
             write_error(error_folder, "jobs", job_id)
         return result
+
+    def import_job(
```

Comment:
Rename to `import` and not `import_job`

Reply:
import is a reserved world by Python, I can't name the function import.
Maybe you know some tricks to do that

Reply:
import_from_json?